core/workflow-java/src/main/java/org/apache/syncope/core/workflow/java/DefaultUserWorkflowAdapter.java

```diff
@@ -207,4 +207,10 @@ public WorkflowFormTO claimForm(final String taskId) {
         throw new WorkflowException(new UnsupportedOperationException("Not supported."));
     }
 
+    @Override
+    public WorkflowResult<String> doRequestCertify(final User user) {
+        //Implement here Identity Recertification Process 
```

Comment:
Form processing is not supported by the default adapter, so remove comment and change message to "Not supported"